apps/demos/Demos/Chat/Customization/jQuery/styles.css

```diff
@@ -0,0 +1,52 @@
+.dx-viewport {
+    padding: 20px;
+    min-width: 780px;
+    border: 1px solid var(--dx-color-border);
+    border-radius: 8px;
+}
+
+.demo-container {
+    min-width: 720px;
+    display: flex;
+    gap: 20px;
+}
+
+.chat-container {
+    display: flex;
+    flex-direction: column;
+    flex-grow: 1;
+    align-items: center;
+    justify-content: center;
+}
+
+.options {
+    padding: 20px;
+    min-width: 280px;
+    background-color: rgba(191, 191, 191, 0.15);
+}
+
+#chat {
+max-width: 480px;
```

Comment:
The formatting is incorrect. I suggest use the `.dx-chat` selector for this purpose, as it allows us to avoid adding an ID to the chat in other frameworks.